libraries/node-core-library/src/Sort.ts

```diff
@@ -231,4 +231,92 @@ export class Sort {
       set.add(item);
     }
   }
+
+  /**
+   * Sort the keys given in an object
+   *
```

Comment:
Nit: missing @param doc comments for each parameter

Reply:
Updated